### PR TITLE
Ft stops

### PR DESCRIPTION
**Pull request by chanelle740**

### CODE REVIEW

**Comment by key-joshua** [comment]
Your pull request is

**Comment by key-joshua** [comment]
Your **Pull Request** is outdated just take an update from **`Develop Branch`** and request reviews

**Comment by chanelle740** [comment]
> Your **Pull Request** is outdated just take an update from **`Develop Branch`** and request reviews

It is up to date

File: src/modules/users/.DS_Store
Comment (Gahamanyi-dev) [inline src/modules/users/.DS_Store:1]: Great work on the PR! Just a quick reminder that tests for every function used are required to ensure the reliability and robustness of the code. Please add these tests and you are good to go.

**Review by yvettegahamanyi — COMMENTED** [review]
Great work on the PR! Just a quick reminder that tests for every function used are required to ensure the reliability and robustness of the code. Please add these tests and you are good to go.

**Review by tuyishimejeandamour — COMMENTED** [review]
resolve the conflict first

**Review by yvettegahamanyi — DISMISSED** [review]

**Review by yvettegahamanyi — DISMISSED** [review]

**Pull request closed without merging**